Formula/c/cfssl.rb

```diff
@@ -25,6 +25,7 @@ def install
 
     system "go", "build", *std_go_args(ldflags:, output: bin/"cfssl"), "./cmd/cfssl"
     system "go", "build", *std_go_args(ldflags:, output: bin/"cfssljson"), "./cmd/cfssljson"
+    system "go", "build", *std_go_args(ldflags:, output: bin/"cfssl-certinfo"), "./cmd/cfssl-certinfo"
     system "go", "build", *std_go_args(ldflags: "-s -w", output: bin/"mkbundle"), "./cmd/mkbundle"
```

Comment:
Can we handle this more robustly, so we don't have to keep adding lines here each time upstream changes what's in the `./cmd` directory?